src/Components/Publishing/Display/__tests__/DisplayAd.test.tsx

```diff
@@ -9,53 +10,91 @@ import "jest-styled-components"
+    const ad = mount(
       <DisplayAd
         adDimension={AdData.adDimension}
         adUnit={AdData.adUnit}
         targetingData={targetingData(StandardArticle, "article")}
+        articleSlug={StandardArticle.slug}
       />
     )
-    .toJSON()
-  expect(component).toMatchSnapshot()
-})
 
-it("renders the component with the correct data and properties in standard layout articles", () => {
-  const ad = mount(
-    <DisplayAd
-      adDimension={AdData.adDimension}
-      adUnit={AdData.adUnit}
-      targetingData={targetingData(StandardArticle, "article")}
-    />
-  )
-
-  const adProps: DisplayAdProps = ad.props()
-  expect(adProps.adDimension).toEqual("970x250")
-  expect(adProps.adUnit).toEqual("Desktop_TopLeaderboard")
-  expect(ad).toHaveLength(1)
-})
+    const adProps: DisplayAdProps = ad.props()
+    expect(adProps.adDimension).toEqual("970x250")
+    expect(adProps.adUnit).toEqual("Desktop_TopLeaderboard")
+    expect(ad).toHaveLength(1)
+  })
+
+  it("renders GPT with the correct properties in standard layout articles", () => {
+    const ad = mount(
+      <DisplayAd
+        adDimension={AdData.adDimension}
+        adUnit={AdData.adUnit}
+        targetingData={targetingData(StandardArticle, "article")}
+        articleSlug={StandardArticle.slug}
+      />
+    )
+
+    const gptComponent = ad.find(GPT)
+    const gptProps: any = gptComponent.props()
+    expect(gptProps.adUnitPath).toEqual("/21805539690/Desktop_TopLeaderboard")
+    expect(gptProps.targeting).toEqual({
+      is_testing: true,
+      page_type: "article",
+      post_id: "594a7e2254c37f00177c0ea9",
+      tags: "Art Market",
+    })
+    expect(gptProps.slotSize).toEqual([970, 250])
+  })
+
+  it("Tracks display ad click", () => {
+    const component = mount(
+      <DisplayAd
+        adDimension={AdData.adDimension}
+        adUnit={AdData.adUnit}
+        targetingData={targetingData(StandardArticle, "article")}
+        articleSlug={StandardArticle.slug}
+      />
+    )
+
+    component.at(0).simulate("click")
 
-it("renders GPT with the correct properties in standard layout articles", () => {
-  const ad = mount(
-    <DisplayAd
-      adDimension={AdData.adDimension}
-      adUnit={AdData.adUnit}
-      targetingData={targetingData(StandardArticle, "article")}
-    />
-  )
-
-  const gptComponent = ad.find(GPT)
-  const gptProps: any = gptComponent.props()
-  expect(gptProps.adUnitPath).toEqual("/21805539690/Desktop_TopLeaderboard")
-  expect(gptProps.targeting).toEqual({
-    is_testing: true,
-    page_type: "article",
-    post_id: "594a7e2254c37f00177c0ea9",
-    tags: "Art Market",
+    expect(trackEvent).toBeCalledWith({
+      context_page: "Article",
+      context_module: "AdServer",
+      context_page_owner_type: "Article",
+      context_page_owner_id: StandardArticle.id,
+      context_page_owner_slug: StandardArticle.slug,
+    })
```

Comment:
We should add an `action_type` prop here to align with [@xtina-starr 's above comments](https://github.com/artsy/reaction/pull/2964/files#r342234920)